install/kubernetes/cilium/values.yaml

```diff
@@ -1680,6 +1685,13 @@ clustermesh:
     # -- Labels to be added to clustermesh-apiserver pods
     podLabels: {}
 
+    # -- PodDisruptionBudget settings
+    # ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
+    podDisruptionBudget:
+      enabled: false
+      maxUnavailable: 1
+      # minAvailable: 1
```

Comment:
K8s only allow set either `maxUnavailable` or `minAvailable`, not both.
![image](https://user-images.githubusercontent.com/6848311/148158533-04d9cbb2-8138-41c8-bf22-e7452939abd2.png)

So, if `maxUnavailable` is set by default, when user provide `minAvailable`, then `maxUnavailable` will NOT be overwriten. So my suggestion is both values is not set by default, and let's user decide which should to go.
```suggestion
    podDisruptionBudget:
      enabled: false
      maxUnavailable:
      minAvailable:
```

Reply:
when podDisruptionBudget is enabled and both the values of minAvailable  and maxUnavailable is not set by default, then will get  error for creating  podDisruptionBudget.

Reply:
Yes. Exact one of `minAvailable` or `maxUnavailable` must be setted when enable PDB.

Or another way, you can set `maxUnavailable: 1` as default, but in `minAvailable`, it should notify user to disable `maxUnavailable` by set `maxUnavailable: null`

Reply:
How can do it? Can you give me an example.

Reply:
If the second way, helm doc should be:
```suggestion
    podDisruptionBudget:
      # -- enable PodDisruptionBudget
      enabled: false
      # -- Minimum number/percentage of pods that should remain scheduled. 
      # When it's set, maxUnavailable must be disabled by `maxUnavailable: null`
      minAvailable: null
      # -- Maximum number/percentage of pods that may be made unavailable
      maxUnavailable: 1
```

Reply:
thanks

Reply:
have done